src/tinypeel/tinypeel.py

```diff
@@ -28,9 +28,11 @@ def runPeelingCycles(pedigree, peelingInfo, args, singleLocusMode=False):
         #     print("Estimating the transmission rate is currently a disabled option")
         # PeelingUpdates.updateSeg(peelingInfo) #Option currently disabled.
 
-        if args.esterrors:
+        if args.est_geno_error_prob:
```

Comment:
@AprilYUZhang should we do an "or" statement here? `if args.est_geno... or args.est_seq...`?

Reply:
I agree, and `if .. or ..:` is better.